core/tests/test_views.py

```diff
@@ -117,6 +164,61 @@ def test_add_comment_unauthorized(client: Client, post: Post) -> None:
     assert reverse("login") in response.url
 
 
+def test_reported_list_post_by_admin(client: Client, admin: User, post: Post, report_data: dict) -> None:
+    client.force_login(admin)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    assert reverse("home") in response.url
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 200
+    reports_count = PostReport.objects.filter(verified=False).count()
+    assert reports_count > 0
+
+
+def test_reported_list_post_by_user(client: Client, user: User, post: Post, report_data: dict) -> None:
+    client.force_login(user)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
```

Comment:
As previously, this checks too many things: post create, etc.. 
Does not check the content of the response. The list might be empty.